src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstancePropertyValue.scala

```diff
@@ -93,7 +93,7 @@ object InstancePropertyValue {
         val numericInstantPropType = v.asNumber.flatMap { jn =>
           val bd = BigDecimal(jn.toString)
           if (jn.toString.contains(".")) { // 1.0 should be a Double not Long
```

Comment:
a bit confused about what old code tried to achieve here :thinking: 
- can decimal separator in string be locale-dependent or it is forced to be in say LC_ALL.C
- should "1e10" be an integer
- would it matter for dms api which numeric type it gets? If yes what exactly it cares about, does it depend on the model types
- is there ever a reason to use narrow types int&float32

Reply:
yes.. wonder if we just as well could just always use double and long.. maybe something to consider..